src/capi/azext_capi/_help.py

```diff
@@ -32,6 +32,9 @@
   - name: --ephemeral-disks -e
     type: string
     short-summary: Use ephemeral disks
+  - name: --kubeadm-commands-file
```

Comment:
users for az capi might not be super familiar with kubeadm so it might make more sense to brand this as more generic "vm  run commands"

Reply:
also we should stay away from using "kubeadm commands" as wording in this PR since we're not allowing the customization of kubeadm itself but what happens _before_ or _after_ kubeadm init/join hence the "pre" and "post"

Reply:
Maybe `--run-vm-commands`?

Reply:
After some offline discussion, we agreed on just `--bootstrap-commands`.